service/src/main/kotlin/io/provenance/explorer/domain/entities/Blocks.kt

```diff
@@ -153,10 +156,12 @@ class BlockProposerRecord(id: EntityID<Int>) : IntEntity(id) {
     companion object : IntEntityClass<BlockProposerRecord>(BlockProposerTable) {
 
         fun save(height: Int, minGasFee: Double?, timestamp: DateTime?, proposer: String?) = transaction {
-            (BlockProposerRecord.findById(height) ?: new(height) {
-                this.proposerOperatorAddress = proposer!!
-                this.blockTimestamp = timestamp!!
-            }).apply {
+            (
+                BlockProposerRecord.findById(height) ?: new(height) {
+                    this.proposerOperatorAddress = proposer!!
+                    this.blockTimestamp = timestamp!!
+                }
+                ).apply {
                 this.minGasFee = minGasFee
             }
```

Comment:
This makes me a sad person...

Reply:
Absolutely! Is this lint's recommendation? If so, maybe there's a way to ignore it